docs/docs/axes/cartesian/timeseries.md

```diff
@@ -22,4 +22,4 @@ var chart = new Chart(ctx, {
 
 ## More details
 
-Please see [the time scale documentation](./time.md) for all other details.
+Please see [the time scale documentation](./time) for all other details.
```

Comment:
I found out the resolution works better when you include the extension.
I'd suggest `./time.mdx`

Reply:
Okay will push the fix